File: C++/Regex_Search.cpp
//Implementation of Regex Search in C++
//Codechef Problem Solving
//Link: https://www.codechef.com/problems/ERROR

#include<bits/stdc++.h>
using namespace std;
const int M = 1e9 + 7;
#define fastio ios_base::sync_with_stdio(false);cin.tie(NULL);
#define int long long int
#define pb push_back
#define mp make_pair
#define d(x) cout<<x<<"\n"


regex b("101");
regex c("010");

int32_t main(void)
{
    int t;
    cin >> t;

    while (t--)
    {
        string x;
        cin >> x;
        smatch m;
        if (regex_search(x, m, b) || regex_search(x, m, c))
            cout << "Good" << endl;
        else
            cout << "Bad" << endl;

        cout << flush;
    }
    return 0;
}

File: Python/Regex Search/Regex_search.py
#   CodeChef Problem
#   https://www.codechef.com/problems/ERROR
#   Using Regex search

import re
Pat1=r"010"
Pat2=r"101"
t=int(input())
while(t):
    x=input()
    if(re.search(Pat1,x) or re.search(Pat2,x)) :
        print("Good")
    else :
        print("Bad")
    t=t-1